documentation/configuration.md

```diff
@@ -13,15 +25,55 @@ metrics: {
     # by this server
     server-id: localhost
 
-    # this section is used to enable/disable JMX reporting
-    reporters: {
+    registries: {
+        "my-application": {
+
+            # specify metrics to include in addition to the default list provided.
+            # The combination of these two lists will be what gets exported to graphite and
+            # other reporters (excluding JMX).
+            metrics-allowed: [
+                "foo.bar"
+            ]
+
+            # the default prefix prepended to metric names on the
+            # `metrics-allowed` list before filtering them is `puppetlabs.<server-id>`. If
+            # this does not match the prefix of an application's metrics, specify a different
+            # prefix here so that metrics will be correctly filtered.
+            metrics-prefix: "my-metrics"
+
+            # enable and disable specific reporters, and set configuration
+            # options specific for the registry.
+            reporters: {
 
-        # enable or disable JMX metrics reporter
-        jmx: {
-            enabled: true
+                # enable or disable JMX metrics reporter
+                JMX: {
```

Comment:
Probably doesn't matter, no, and yeah it came in through copypasta.  I can fix that up in the docs.